src/transformers/models/bigscience176b/modeling_bigscience176b.py

```diff
@@ -326,8 +391,35 @@ def get_slopes_power_of_2(n):
         # tp_world_size = mpu.get_tensor_model_parallel_world_size()
         # tp_index = mpu.get_tensor_model_parallel_rank()
         # alibi = alibi.reshape((tp_world_size, -1, *alibi.shape[1:]))[tp_index]
+        alibi = alibi.to(dtype)
 
         return alibi
+    def forward_attention(
```

Comment:
where do we use `forward_attention`?

Reply:
I used to test it during my first tests, removed it [here](https://github.com/younesbelkada/transformers/commit/39644e770ca8cbc8315d1fbce0c19d1e41097c15)